### PR TITLE
Add docker-compose script

### LINKED ISSUE #22: Add docker-compose config to spin-up local plug network

### PR DESCRIPTION
**Pull request by jordy25519**
Closes #22 

Full credit to @elim0322, just porting his work into this repo

### CODE REVIEW

**Comment by jordy25519** [comment]
Stopped the CI as there's no code change

File: docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+            - --validator
+    node-5:
+        image: plugnet/plugblockchain:1.0.0-rc1
+        container_name: node-5
+        volumes: ["./data/node-5:/mnt/node"]
+        command:
+            - /usr/local/bin/substrate
+            - --base-path=/mnt/node
+            - --chain=dev
+            - --bootnodes=/dns4/node-1/tcp/30333/p2p/QmQmJTi2eDg53Ws3N5XaixMmiGCmhNNJX7bPBMirnD2JEs
```
Comment (cowboy-bebug) [inline docker-compose.yml:65]: do we want to add `--reserved-node` as well?

**Review by cowboy-bebug — APPROVED** [review]
Looks good thanks!

File: docker-compose.yml (hunk 1 above)
Comment (jordy25519) [reply docker-compose.yml:65]: I don't think it's needed for a testnet

**Pull request merged**